appveyor.yml

```diff
@@ -1,22 +1,57 @@
 version: 1.0.{build}
+os:
```

Comment:
all changes of this PR will go into the master once this PR gets merged, so it's not possible to ignore some parts of it.

It's probably best to remove the changes to the .yml in this file, and submit them all in the 2nd one.